src/components/Modals/CreateItemModal/CreateItemModal.tsx

```diff
@@ -403,7 +403,12 @@ export default class CreateItemModal extends React.PureComponent<Props, State> {
       }
     } else {
       const url = URL.createObjectURL(contents[model])
-      const { image, info } = await getModelData(url, { width: 1024, height: 1024 })
+      const { image, info } = await getModelData(url, {
+        width: 1024,
+        height: 1024,
+        extension: model.split('.').pop(),
```

Comment:
I think we can use `getExtension` from file.ts both here and in line 435 below if you think is useful

We can probably replace a few more places with that method, but did a quick grep on the codebase and could not find anything